index.html

```diff
@@ -37,7 +37,160 @@ <h1>I'm Newton <br>Glad to see you!</h1>
+                        </li>
+                    </ul>
+                    <a href="" class="project-links">See Project</a>
+                </div>
+            </li>
+            <li class="categories">
+                <img src="./img/portfolio-stories2.png" alt="Tonic Category Image">
+                <div class="left-block">
+                    <h2 class="project-title">
+                        Multi-Post Stories
+                    </h2>
+                    <span class="canopy">
+                        CANOPY
+                        <ul>
+                            <li><img src="./img/counter.png" alt=""></li>
+                            <li>Back End Dev</li>
+                            <li><img src="./img/counter.png" alt=""></li>
+                            <li>2015</li>
+                        </ul>
+                    </span>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="portfolio-tags">
+                        <li>
+                            html
+                        </li>
+                        <li>
+                            css
+                        </li>
+                        <li>
+                            javascript
+                        </li>
+                    </ul>
+                    <a href="" class="project-links">See Project</a>
+                </div>
+            </li>
+        </ul>
+    </section>
+    <section class="about-myself">
+        <h1>About Myself</h1>
+        <p >Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <span>
+            <p class="lets-connect">LET'S CONNECT</p>
+        
+        <ul class="social-icons">
+            <li>
+                <a href="https://github.com/flemton" target="_blank"><img src="./img/github-icon.png" alt="GitHub Profile"></a>
+            </li>
+            <li>
+                <a href="https://www.linkedin.com/in/alhassan-o-83039a80/" target="_blank"><img src="./img/linkedin-icon.png" alt="linkedin Profile"></a>
+            </li>
+            <li>
+                <a href="https://web.facebook.com/n3wtongh/?_rdc=1&_rdr"><img src="./img/Vector.png" alt="Facebook Profile" width="18px" height="20px"></a>
+            </li>
+            <li>
+                <a href="https://twitter.com/oalhassan847" target="_blank"><img src="./img/twitter-icon.png" alt="linkedin Profile"></a>
+            </li>
+            <li>
+                <a href="https://medium.com/" target="_blank"><img src="./img/medium-icon.png" alt="Medium Page"></a>
+            </li>
+        </ul>
+        <a href="" class="project-links">Get my resume</a>
+        </span>
+        <span class="skills-list">
+            <img src="./img/skills-list.png" alt="">
+        </span>
```

Comment:
- [ ] Using an image tag to represent this whole section is not a correct implementation 👇 

![image](https://user-images.githubusercontent.com/78251927/205413203-0106efe6-5fd8-40f5-9c26-64d0475d66bf.png)

- [ ] As per [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`. I can see only used the image on Figma for the entire section. 

- [ ] Please kindly check the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) again to fix this 🙏.